### ISSUE TITLE
For a given envoy commit, how to find the corresponding go control plane commit?

### ISSUE THREAD

**Issue by VivekSubr** [envoyproxy/go-control-plane#790]
For example, istio 1.19 uses envoy commit 47297e26f07520d39272e5925ac1fee05f50ced3, how to find out the go control plane that corresponds to this commit?

**Comment by sunjayBhatia** [comment]
we have some internal scripting to do so at the moment that checks out the envoy source at the desired version and runs:

```
grep '"envoy-build-tools"' bazel/repository_locations.bzl -A 3 | grep version | cut -d'"' -f2
```

**Comment by sunjayBhatia** [comment]
Oh sorry, I'm mixing up two totally different things, I gave you a command for finding the envoy-build-tools commit

**Comment by sunjayBhatia** [comment]
What I usually do to correlate the envoy -> go-control-plane version is:
- check out envoy version
- run `git log api` and record the latest commit sha
- move over to the go-control-plane repo, make sure it is up to date with main
- run `git log` and search for the commit sha you recorded above in commit messages

Been doing it manually but can be scripted

**Comment by phlax** [comment]
the script ot hceck is `ci/sync_envoy.sh` i think - but its kinda hard to follow in terms of commits- i think the gist is that only commits when protos change so probs the relationship isto changes in `api/`

just read above response - probs more accurate

**Comment by VivekSubr** [comment]
Thanks, will do!
